app/assets/stylesheets/views/comments.scss

```diff
@@ -275,3 +275,59 @@
 .moderator-template-button:not(.active) {
   @extend .crayons-btn--outlined;
 }
+
+#toggle-comments-sort-dropdown {
+  cursor: pointer;
+}
+
+#comments-sort-dropdown-container {
+  z-index: var(--z-dropdown);
+
+  .comments-sort-dropdown {
+    left: 0;
+    padding: var(--su-4);
+    padding-right: var(--su-6);
+    width: 310px;
+
+    &__list {
+      list-style-type: none;
+    }
+  }
+
+  .comment-sort-option {
+    margin-top: var(--su-2);
+    padding-left: var(--su-7);
+
+    &__header {
+      color: var(--base-90);
+      font-weight: var(--fw-medium);
+      display: block;
+    }
+
+    .sort-selected {
+      flex-basis: auto;
+      min-width: 40px;
+    }
+
+    .comment-sort-option__header {
+      .crayons-icon {
+        display: none;
+        position: absolute;
+        left: var(--su-4);
+        color: var(--link-branded-color);
+      }
+
+      &[aria-current='page'] {
+        font-weight: var(--fw-bold);
+
+        .crayons-icon {
+          display: block;
+        }
+      }
+    }
+  }
+
+  .comments-sort-title {
+    margin-bottom: var(--su-3);
+  }
```

Comment:
refactor (blocking) - as there's only one property here, let's favour using helper utility classes on the element itself. You should be able to use `mb-3` classname on the element to get the same effect